### PR TITLE
fetch button centered

### PR DESCRIPTION
**Pull request by satyaarthchhabra**
hi in this pull request i have tried to resolve this  [issue ](https://github.com/bench-routes/dashboard/issues/31). 

description:
i have just aligned the items both vertically and horizontally

### CODE REVIEW

**Comment by ruddi10** [comment]
@satyaarthchhabra Its a good practice of adding proof of what you have achieved while making the PR, A small video or gif will be good for eg- use Peek

File: src/components/TimeQuerier/index.tsx [hunk 1]
```diff
@@ -132,7 +132,14 @@ const TimeQuerier: React.FC = () => {
           </NumberInputStepper>
         </NumberInput>
       </Box>
-      <Box d="flex" pt="2" pb="1" w="100%">
+      <Box
+        d="flex"
+        alignItems="center"
```
Comment (aquibbaig) [inline src/components/TimeQuerier/index.tsx:137]: I think we are better off with a `textAlign: center` property. A flexbox with one child (Button) does not make sense to me

**Review by aquibbaig — CHANGES_REQUESTED** [review]

**Comment by aquibbaig** [comment]
@satyaarthchhabra any updates yet?

**Comment by rijusougata13** [comment]
hi @aquibbaig  can I go with this issue if no one working on it?

**Comment by aquibbaig** [comment]
@rijusougata13 I think we should move ahead. This PR is like 28days old and doesn't have any updates. Please create a new PR with the changes. I'm closing this for now, we'll reopen it if the author comes back to it

**Pull request closed without merging**